src/pages/studyView/StudyViewUtils.tsx

```diff
@@ -778,9 +854,14 @@ export function getGenericAssayChartUniqueKey(
 const UNIQUE_KEY_SEPARATOR = ':';
 
 export function getUniqueKeyFromMolecularProfileIds(
-    molecularProfileIds: string[]
+    molecularProfileIds: string[],
+    chartType?: ChartTypeEnum
 ) {
-    return _.sortBy(molecularProfileIds).join(UNIQUE_KEY_SEPARATOR);
+    let ids = _.sortBy(molecularProfileIds);
+    if (chartType === ChartTypeEnum.STRUCTURAL_VARIANTS_TABLE) {
```

Comment:
is this to distinguish the different types of SV charts?   i think this needs a comment.  also, if this ID is chart centric, maybe we change the name of the function to reflect that.  Since it not clear by chartType should be an input based on the current name

Reply:
Indeed. The unique table key for a chart was derived from the molecular profile name(s) of the data shown in that table. 

1. This method is chart centric as you mention. I propose to rename this method to:

`getUniqueChartKeyFromMolecularProfileIds`

2. ChartType is an optional input. It is only used in my implemention for the new SV table because when different tables use the same data (same molecular profile ids), this strategy no longer works. However, I think that we better incorporate the CartTTypeEnum in the key like so:

```
export function getUniqueChartKeyFromMolecularProfileIds(
    molecularProfileIds: string[],
    chartType?: ChartTypeEnum
) {
    let ids = _.sortBy(molecularProfileIds);
    let returnValue = ids.join(UNIQUE_KEY_SEPARATOR);
    return chartType ? chartType + UNIQUE_KEY_SEPARATOR + returnValue :  returnValue;
}
```

3. How would you feel about making `chartType` a required param? This way is is also clear to developers in the future that it is chart centric.